ct-app/tests_endurance/__main__.py

```
@@ -7,11 +8,11 @@
 from . import EnduranceTest
 
 
-def set_envvars(pairs: list[tuple[str, str]]):
-    if isinstance(pairs, tuple):
-        pairs = [pairs]
+def set_envvars(pair: tuple[str, str]):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)